tests/e2e/helpers/base.js

```diff
@@ -104,7 +104,7 @@ export function runBaseTests (br) {
     await t.click(nav.desktop.mode1Up);
     const onePageViewContainer = await Selector('.BRpageview');
     await t.expect(onePageViewContainer.visible).ok();
-    const images = onePageViewContainer.find('.BRpagediv1up');
+    const images = onePageViewContainer.find('.BRmode1up');
```

Comment:
make it clearer that a page is being selected
```suggestion
    const images = onePageViewContainer.find('.BRpagecontainer.BRmode1up');
```